draw_flip_book/lib/main.dart

```diff
@@ -114,20 +137,36 @@ class _FlipBookPageState extends State<FlipBookPage> {
       child: FloatingActionButton(
         onPressed: () {
           setState(() {
-            // TODO: loop all the frames in sequence
+            _startAnimation();
           });
         },
         child: Icon(Icons.play_arrow),
       ),
     );
+    final stopButton = Container(
+      child: FloatingActionButton(
+        onPressed: () {
+          setState(() {
+            // Add "null" to the points to avoid a line being drawn upon
+            // post animation paint attempts
+            points1..add(null);
```

Comment:
This can just be points1.add(null);
.. returns the points1 object but you don't need it.

Reply:
true, cleaned up all occurrences of that